static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -243,7 +243,11 @@ function ItemTitle({item, searchSubstring, isChild}: ItemTitleProps) {
 
   return (
     <SearchItemTitleWrapper>
-      {!isFirstWordHidden && <FirstWordWrapper>{firstWord}</FirstWordWrapper>}
+      {!isFirstWordHidden && (
+        <FirstWordWrapper>
+          {firstWord} {item.titleBadge}
+        </FirstWordWrapper>
+      )}
```

Comment:
What's this change for?

Reply:
if you just open the dropdown it doesn't render the badge
![image](https://user-images.githubusercontent.com/7349258/234671775-d6a38e29-610f-4b68-8a56-ab298f0e4489.png)

you'd have to search for something that matches "click"
![image](https://user-images.githubusercontent.com/7349258/234671837-a4d97503-776f-426e-b9da-6d5c844c0fb0.png)


i missed this the first time around

Reply:
nit: I would make this two PRs, since the title of this PR doesn't mention any bug fixes and these two changes are functionally unrealted

https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84

Reply:
cool, i'll separate the two.